pkg/bpftrace/build.yml

```diff
@@ -0,0 +1,8 @@
+image: bpftrace
+network: true
+arches: 
+  - amd64
+config:
```

Comment:
This can be dropped if if the main consumption model is to drop the binary into the root filesystem